app/views/statistics/data.html.erb

```diff
@@ -93,21 +102,59 @@
     </tbody>
   </table>
 
+  <h4>Summary usage statistics by country</h4>
+  <p>
+  The summary usage statistics by country data groups usage statistics by <code>Country</code>.
+  </p>
+  <ul>
+    <li>Where registered users have not yet nominated their country, their statistics are grouped under "Not specified".</li>
+    <li>Where the number of assessments for a country is below the privacy threshold, statistics are grouped under "All other countries".</li>
+  </ul>
+
+  <table class="table">
+    <thead>
+      <tr><th>Field</th><th>Definition</th></tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><code>Country</code></td>
+        <td>A country name or a grouping where country isn't specified or would risk disclosing individual results.</td>
+      </tr>
+      <tr>
+        <td><code>Registered users</code></td>
+        <td>Number of users registered on the website.</td>
+      </tr>
+      <tr>
+        <td><code>Users with organisations</code></td>
+        <td>Number of users that have specified an organisation.</td>
+      </tr>
+      <tr>
+        <td><code>Completed assessments</code></td>
+        <td>Total number of completed assessments. A user can have multiple completed assessments.</td>
+      </tr>
+      <tr>
+        <td><code>Total assessments</code></td>
+        <td>Total number of assessments. A user can have multiple assessments, including one in-progress and several completed.</td>
+      </tr>
+    </tbody>
+  </table>
+
   <h4>Aggregate statistics</h4>
 
   <p>
-  Both of the aggregate statistics files have the same content but aggregate statistics for different collections of organisations.
+  The aggregate statistics files have similar content but aggregate statistics for different collections of organisations or group statistics by country.
   </p>
 
   <ul>
-    <li>"All organisations" refers to all users of the system</li>
+    <li>"All organisations" refers to all users of the system.</li>
+    <li>"All organisations by country" refers to all users of the system, grouped by country.</li>
```

Comment:
I believe the code is only reporting on users with an organisation so the text perhaps should be 
_"All organisations by country" refers to all users with an organisation, grouped by country._